Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM registry.ci.openshift.org/openshift/release:golang-1.18 AS builder
+FROM registry.ci.openshift.org/openshift/release:rhel-8-release-golang-1.19-openshift-4.12 AS builder
```

Comment:
I see in other Dockerfile it is `registry.ci.openshift.org/ocp/builder:rhel-8-golang-1.19-openshift-4.12`, it will be great if you can point me to any doc or link from where we are picking this image entry.

Reply:
We used to have golang builder as `registry.ci.openshift.org/openshift/release:golang-1.18` for CI jobs but i found that  `registry.ci.openshift.org/openshift/release:golang-1.19` does not exist.
From [slack thread ](https://coreos.slack.com/archives/C039LFF79K6/p1664290759245199?thread_ts=1661427230.036119&cid=C039LFF79K6)I learnt that `registry.ci.openshift.org/openshift/release:rhel-8-release-golang-1.19-openshift-4.12` is what officially supported by ART team and they are kept upto date.

Regarding this docker file I am not sure where it will be used but to be consistent I have updated it.